scripts/npm/unlink/index.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var path = require( 'path' );
+var fs = require( 'fs' );
+var componentsDir = path.join( __dirname, '..', '..', '..', 'components' );
+var components = [ 'cf-buttons',
```

Comment:
We could probably pull the list programmatically from the package.json file. I'll look into making this improvement.